test/upstream-e2e-tests.sh

```diff
@@ -26,7 +26,7 @@ fi
 # Run upgrade tests
 if [[ $TEST_KNATIVE_UPGRADE == true ]]; then
   install_serverless_previous
-  run_rolling_upgrade_tests "${UPGRADE_TEST_SCOPE:-serving,eventing}"
```

Comment:
Do we want to drop this flag? I personally haven't used it.

Reply:
Yeah, I think we want to drop it. The variable is not defined anywhere else (e.g. vars.bash) so this change should be sufficient.

Reply:
Okay.